sentences/pl/_common.yaml

```diff
@@ -33,6 +33,12 @@ lists:
       type: "percentage"
       from: 0
       to: 100
+  brightness_level:
+    values:
+      - in: (max[a|imum] | maks[a|imum])
```

Comment:
What about forms like `maksymalną` and `minimalną`? Wouldn't that more natural to ask 'Ustaw jakość lampy na maksymalną' instead of 'Ustaw jasność lampy sufitowej na maksimum'?